docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2387,12 +2387,11 @@ Rules with `$replace` modifier are supported by AdGuard for Windows, Mac, and An
 
 #### **`urltransform`** {#urltransform-modifier}
```

Comment:
Maybe it could be changed to:
```suggestion
#### **`$urltransform`** {#urltransform-modifier}
```
The same like it's done in the case of other modifiers.